modules/test/playwright/pages/portal-workflow-kaleo-designer-web/DiagramViewPage.ts

```diff
@@ -30,6 +34,10 @@ export class DiagramViewPage {
 		await this.sourceViewButton.click();
 	}
 
+	async publishWorkflowDefinition() {
+		await this.publishWorkflowDefinitionButton.click();
+	}
```

Comment:
Hey Carlos, similar to the previous comment. I was unable to visualize the benefits of this function, I believe we can just call the locator outside the page.